File: API-Angular/Models/StudentCRUDDTOs/StudentCreateDTO.cs
﻿namespace API_Angular.Models.StudentCRUDDTOs;

public class StudentCreateDTO
{
    public int StudentId { get; set; } // just practise
    public string Name { get; set; }
    public string DateOfBirth { get; set; }
    public bool Passed { get; set; } // checkbox
    public string Gender { get; set; } // radio

    public int CountryId { get; set; }

    public List<SubjectCreateDTO> Subjects { get; set; } // sk..

}

File: Client-Angular/src/app/admin/models/StudentCreateDTO.ts
import { SubjectCreateDTO } from "./subjectCreateDTO";

export class StudentCreateDTO {

  studentId: number | any; // just practise
  name: string | any;
  dateOfBirth: string | any;
  passed: boolean | any;  // checkbox ... act
  gender: string | any;   // radio ... sta

  countryId: number | any;    // cli...

  subjects: SubjectCreateDTO[];

  
  constructor() {
    this.studentId = null;
    this.name = null;
    this.dateOfBirth = null;
    this.passed = true;
    this.gender = null;

    this.countryId = null;

    this.subjects = [];

  }

}
